tests/github/test_dal.py

```diff
@@ -23,8 +23,24 @@ def test_org_security(mocker: MockerFixture):
         }
     }
     mocker.patch("checkov.common.vcs.base_vcs_dal.BaseVCSDAL._request_graphql", return_value=mock_data)
-    result = dal.get_organization_security()
-    assert result
+    result1 = dal.get_organization_security()
+
+    mock_data2 = {
```

Comment:
can you create a separate test function for this case and name it accordingly, something like `test_org_security with_empty_ description`.

Reply:
thanks :) changed it!